### PR TITLE
Added DC Universe Online

### PR DESCRIPTION
**Pull request by Zemogiter**

### CODE REVIEW

File: Applications/Games/DC Universe Online/Online/script.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+print("Debug output");
```
Comment (plata) [inline Applications/Games/DC Universe Online/Online/script.js]: please remove debug print outs

File: Applications/Games/DC Universe Online/Online/script.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  	"scriptName": "Online",
+  	"compatibleOperatingSystems": [],
```
Comment (plata) [inline Applications/Games/DC Universe Online/Online/script.json]: `testingOperatingSystems` must be a subset of `compatibleOperatingSystems`

File: Applications/Games/DC Universe Online/Online/script.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+new OnlineInstallerScript()
+    .name("DC Universe Online")
+    .editor("Sony Entertainment")
+    .applicationHomepage("http://www.dcuniverseonline.com/")
+    .author("Zemoscripter")
+    .url("https://launch.daybreakgames.com/installer/DCUO_setup.exe")
+    .category("Games")
+    .executable("Origin.exe")
+    .wineDistribution("upstream")
```
Comment (plata) [inline Applications/Games/DC Universe Online/Online/script.js]: "upstream" is default, so no need to write it here

File: Applications/Games/DC Universe Online/Online/script.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+include(["Engines", "Wine", "Verbs", "d3dx9"]);
+
+new OnlineInstallerScript()
+    .name("DC Universe Online")
+    .editor("Sony Entertainment")
+    .applicationHomepage("http://www.dcuniverseonline.com/")
+    .author("Zemoscripter")
+    .url("https://launch.daybreakgames.com/installer/DCUO_setup.exe")
+    .category("Games")
+    .executable("Origin.exe")
```
Comment (plata) [inline Applications/Games/DC Universe Online/Online/script.js]: Does it use EA Origin or is the executable really called "Origin.exe"?

**Comment by plata** [comment]
If you have tested it and everything works, I can approve this.

**Comment by Zemogiter** [comment]
I've tested it and it ran just fine but the game crashes sometimes and even with the print debug command I am getting no clue why.

**Comment by plata** [comment]
As it's marked as "testing", that's ok.

**Pull request merged**